memberslist.json

```
@@ -9,6 +9,7 @@
         {"name": "Jasper Thomas-Yount", "href": "member.html?name=thomas-yount"},
         {"name": "Ben Snyder", "href": "/member.html?name=snyder"},
         {"name": "Yen Mai", "href": "/member.html?name=mai"},
-	{"name": "Ezra Crowe","href": "/member.html?name=Ezra-Crowe"}
+	{"name": "Ezra Crowe","href": "/member.html?name=Ezra-Crowe"},
+	{"name": "Kai Monno","href": "/member.html?name=KaiMonno"}
```

```suggestion
	{"name": "Kai Monno","href": "/member.html?name=kai"}
```

In order for the link to work right, the `name=` part here needs to be the same as in the filename. Since your file is called `kai.json`, you could change this to `name=kai`, or you could change the name of your file.